tests/test_languages.py

```diff
@@ -1605,6 +1611,10 @@ def test_translation(self, shortname, datetime_string, expected_translation):
         param('uk', "18 хвилин тому", "18 minute ago"),
         param('uk', "через 22 роки", "in 22 year"),
         param('uk', "цього тижня", "0 week ago"),
+        param('uk', "півгодини тому", "30 minute ago"),
+        param('uk', "пів години тому", "30 minute ago"),
+        param('uk', "півроку тому", "6 month ago"),
```

Comment:
6 month**S** ago

Reply:
Please look at other test cases for all other languages. The goal of the library is not to translate dates into grammatically correct English sentences, it's just a convenient step before converting them into datetime objects.